qemu/tests/cfg/single_driver_install.cfg

```diff
@@ -26,6 +26,10 @@
                 backend_type = passthrough
                 filename_passthrough = /dev/random
         - with_viostor:
+            # For latest windows guests, changing system disk driver between
+            # virtio_blk and virt_scsi will cause BSOD "INACCESSABLE_DEVICE".
+            # Do a workaround here to avoid such scenario.
+            no virtio_scsi
             driver_name = viostor
             device_name = "Red Hat VirtIO SCSI controller"
             drive_format_image1 = ide
```

Comment:
1. only virtio_blk maybe more clear?
2. suggest to remove "drive_format_image1 = ide"
3. we can add variants
e.g
- with_viostor:
    - virtio_blk:
    - virtio_scsi:
     driver_name = vioscsi
     device_name = "Red Hat VirtIO SCSI pass-through controller"

the same as with_vioscsi